content/docs/migration_1.0/Concepts.md

```diff
@@ -0,0 +1,126 @@
+
+## Query & Queryable
+
+`Query` and `Queryable` have been slightly reworked.  
+For the API replying to a `Query` from a `Queryable` declared on a session: 
+The `reply` function has been split into 3 separate functions variants depending on the type of reply the user wants to send.  
+`reply` , `reply_del`, `reply_err`  
+
+We have added the ability to get the underlying `Handler` of a Queryable as well.
+
+## Use accessors to get private members
+Across language bindings we encapsulate members of structs, and they can’t be accessed directly now.  
+The only way to access struct values is to use the getter function associated with them.
+
+
+## Pull Subscribers have been removed
+
+The concept of a pull subscriber no longer exists in Zenoh,
+However, when creating a `Subscriber`, it may be the case that developers only care about the latest data and want to discard the oldest data. 
+We can use `RingChannel` to get this behaviour.
+This contrasts a `FIFOChannel` which is the default channel type used internally in Subscribers
+You can take a look at examples of usage in any language’s examples/z_pull.x
+
+## Timestamps
+
+We now tie generating a timestamp to a Zenoh session, with the timestamp inheriting the `ZenohID` of the session.
+
+This will affect user-created plugins and applications that need to generate timestamps in their Storage and sending of data.  
+⚠️ Note: Timestamps are important for Storage Alignment and Replication. Data stored in Data bases must include a Timestamp to be properly aligned across Data Stores by Zenoh. 
+The `timestamping` configuration option must also be enabled for this.
+
+## Storages
+The storage-manager will fail to launch if the timestamping configuration option is disabled.  
+From Zenoh 1.0.0 user-applications can load plugins.  
+A, somehow, implicit assumption that dictated the behaviour of storages is that the Zenoh node loading them **has to add a timestamp to any received publication that did not have one**. This functionality is controlled by the `timestamping` configuration option.  
+Until Zenoh 1.0.0 this assumption held true as only a router could load storage and the default configuration for a router enables `timestamping`. However, in Zenoh 1.0.0 nodes configured in `client` & `peer` mode can load storage and *their default configuration disables `timestamping`*.
+
+## Config Changes
+
+### Plugin Loading
+
+We added the ability for user-applications to load compiled plugins written in Rust, regardless of  language ! 
+
+Usage of this feature is achieved by simply enabling the `plugins_loading` section in config file with the members `enabled` set to true, and specifying the `search_dirs` for the plugins. 
+
+If no search directories were specified, then the default search directories are 
+`".:~/.zenoh/lib:/opt/homebrew/lib:/usr/local/lib:/usr/lib”` 
```

Comment:
The behaviour/config is expected to be modified by @fuzzypixelz . This guide should be update accordingly once the PR lands.

Reply:
As promised, see PR1278 on eclipse-zenoh/zenoh.

Reply:
Linking: https://github.com/eclipse-zenoh/zenoh/pull/1278